apps/www/src/app/(public pages)/hiring-managers/learn/(pages)/about-digital-talent.md

```diff
@@ -0,0 +1,50 @@
+---
+title: About the Digital Talent Attraction and Development Team
+description: British Columbia, BC Government, hiring servies, Office of the Chief Information Officer, about us, Digital Talent Attraction and Development Team, DTAD
+---
+
+On April 6th, 2023, the Digital Office released [BC Government's Digital Plan](https://www2.gov.bc.ca/assets/gov/data/digital-government/digital-plan.pdf). One of many items reported on are hiring barriers:
+
+
+> Early on, the Exchange Lab recognized that ministries faced several barriers to building digital capacity. It often took months or years to bring in external vendors to support digital initiatives. Further, there was no clear path for existing employees to develop their own digital skills, and ministries struggled to attract and hire talented technologists from outside government.
+
+
+Legacy policies prevented seamless hiring and this impacted the public. Modernizing takes innovation, of both culture and process. Our team, the Digital Talent Attraction and Development Team (DTAD), was one of the responses to these barriers. 
+
+
+DTAD is a subject matter expert with cross-functional talent from the Office of the Chief Information Officer (OCIO) and the Public Service Agency (PSA). We contribute to resolutions of the Province’s complex problems by designing services to build internal capacity for digital talent across government. The team partners across multiple program areas across BC Government to tap into multiple subject matter experts on digital talent.
+ 
+
+We work, live, and play on the lands of lək̓ʷəŋən People, now known as the Esquimalt and Songhees Nations, scəw̓aθən (Tsawwassen), xʷməθkʷəy̓əm (Musqueam) and Coast Salish Peoples, Lheidli T'enneh, and Tk’emlúps te Secwépemc territory. 
+
+
+
+The diversity of these lands shapes our team members and impacts the foundation of our hiring services. 
+
+
+Across Provincial Government from November 2021 – December 2022, we: 
+
+- Completed sixteen competitions 
+- Interviewed one hundred and seventy five candidates 
+- Had five hundred and eighty four Applicants
+    - four hundred and ninety external to B.C. Government
+    - ninety four internal to B.C. Government
+

```

Comment:
T mentioned using numerals in place of the spelled out numbers, or, using th spelled out version and numbers in (), either before or after.

Reply:
Connected with J earlier today, and we will stick with numerical values. I've changed the file to reflect this.